src/test/resources/application-test.yml

```
@@ -1,4 +1,4 @@
 spring:
   data:
     neo4j:
-      database: neo4j
+      database: neo4j
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)